src/daemon/dlt_daemon_common.c

```diff
@@ -1370,11 +1393,12 @@ int dlt_daemon_user_send_log_level(DltDaemon *daemon, DltDaemonContext *context,
                  errno != 0 ? strerror(errno) : "Unknown error");
 
         if (errno == EPIPE) {
-#ifndef DLT_USE_UNIX_SOCKET_IPC
-            /* Close connection */
-            close(context->user_handle);
-#endif
-            context->user_handle = DLT_FD_INIT;
+            /* TODO: Verify that this is OK. That is, a context with a user_handle will always be
+               for local ECU with id == daemon->ecuid and there will always be an application for
+               a context that is active. If not, NULL check needed here or in
+               dlt_daemon_application_reset_user_handle().*/
+            app = dlt_daemon_application_find(daemon, context->apid, daemon->ecuid, verbose);
+            dlt_daemon_application_reset_user_handle(daemon, app, verbose);
```

Comment:
Perhaps add a NULL check here (even though I am not certain app can ever be NULL)?

Reply:
I added a NULL check here now (since I am not sure, better to have NULL check in our branch for now).

Leaving issue open to potentially remove it.